rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_07_22_ckusdt.md

```diff
@@ -0,0 +1,43 @@
+# Proposal to upgrade the ledger suite orchestrator canister to add ckUSDT
```

Comment:
@manudrijvers @THLO It might actually be possible to change to title of the proposal using [this option](https://github.com/dfinity/ic/blob/5c1ddd3bf1fd2c29a3b9fab3e4ce368e5fa0a984/rs/registry/admin-derive/src/lib.rs#L78). So how about "Add ckUSDT" for a title?

Reply:
I like the idea of adding a title. How about adding a bit more information, e.g., "Adding the ckUSDT ledger suite"?

Reply:
Too late for ckUSDT 🙈 (I submitted [this](https://dashboard.internetcomputer.org/proposal/131374) proposal this morning). But I think what you suggested makes sense for the next proposals.

Reply:
Thinking about it again (for [ckWSTETH](https://github.com/dfinity/ic/pull/612)), I'm thinking that "ledger suite" is more an implementation detail, what we are adding is a new ckERC20 token and part of that is creating the corresponding ledger suite, but another part is for example letting the minter know that there is a new token. So I will reformulate a bit.